Source/ToolCore/Assets/AssetDatabase.h

```diff
@@ -56,6 +56,7 @@ class AssetDatabase : public Object
     void Scan();
 
     void ReimportAllAssets();
+    void ReimportAllAssetsInDirectory(const String& directoryPath);
```

Comment:
This could probably use a recursive flag, and now that I see ReimportAllAssets above it, looks ok to me